doc/content/getting-started/aws/ecs/troubleshooting/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Troubleshooting AWS ECS Deployment"
+description: ""
+weight: 3
+---
+
+This section contains information to troubleshoot {{% tts %}} deployment.
+
+### ECS pulled wrong image version
+
+The image tags starting with `3.x` contain the latest patch version. For this reason, you should use images starting with `3.x.y`, where you specify the desired patch version yourself. Otherwise, when for some reason an ECS task restarts, it can pull the `3.x` image again, which could be in the meantime updated to a new patch version.
+
+### Identity Server fails with `error:pkg/identityserver/store:database (database error)`
+
+This error is usually caused by a network failure. In this case, and in case of similar connection errors, we recommend:
+1. Restarting relevant ECS tasks
+2. Waiting for all startup processes to complete, as sometimes certain elements might start in wrong order
+
+If this doesn't help, please start troubleshooting the network connection (for example, in this case between the Identity Server and the Aurora database).
+
+### The session information is lost for all devices, and the uplinks are dropped by the Network Server
+
+While intuitively ElastiCache seems to be just cache, in fact it is a fully capable database, and {{% tts %}} uses it as such. In particular, among other things, Network Server uses it for storing the session information for devices. If there is any issue with the Redis database, it causes loss of devices session data.
+
```

Comment:
I'd rather that they look at Elasticache and/or Stack metrics and prevent the problem rather than restore later.

Can we add some metrics information here?

Reply:
Well, uh, the point of this section is to tell people "ElastiCache isn't just cache for us. If you wiped it, you need to restore from backup".